<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sim1Viewer</class>
 <widget class="QDialog" name="Sim1Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UAV Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
  color: #ccc;
}

QDialog {
	background: black;
}

QFrame {
  border-color: #666;
}

QCheckBox, QCheckBox::indicator, QCheckBox::indicator::checked, 
QCheckBox::indicator::unchecked {
	background: none;
}

QLineEdit, QDoubleSpinBox {
  background: #222;
  border-color: #666;
}

QPushButton, QToolButton {
  background: #222;
  border-color: #666;
}

QPushButton::disabled {
  color: #666;
  background: #111;
}

QPushButton::enabled {
  background: #222;
  color: #ccc;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="uav::viewer::RenderWidget" name="glPanel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmInfo">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtElevation"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Altitude:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Velocity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="txtVelocity"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Pulse Rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="txtPulseRate"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frmSettings">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Terrain File</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="btnTerrainFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblGimbalAngle">
        <property name="text">
         <string>Gimbal Static Angle (Â°)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spnGimbalAngle"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="txtTerrainFile"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chkShowSettings">
       <property name="text">
        <string>Show Settings</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkShowInfo">
       <property name="text">
        <string>Show Info</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkShowTerrain">
       <property name="text">
        <string>Show Terrain</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>uav::viewer::RenderWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>src/viewer/renderwidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>btnTerrainFileClicked()</slot>
  <slot>terrainFileChanged(QString)</slot>
  <slot>btnCloseFormClicked()</slot>
 </slots>
</ui>
